data/src/main/java/com/sopt/data/dto/response/ResponseGetGroupOngoingDto.kt

```diff
@@ -0,0 +1,33 @@
+package com.sopt.data.dto.response
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ResponseGetGroupOngoingDto(
+    @SerialName("groupOngoingInfo") val groupOngoingInfo: GroupOngoingInfoDto,
+    @SerialName("ongoingAppointments") val ongoingAppointments: List<OngoingAppointmentDto>
+)
+
+@Serializable
+data class GroupOngoingInfoDto(
+    @SerialName("groupName") val groupName: String,
+    @SerialName("groupProfileImageUrl") val groupProfileImageUrl: String? = null,
+    @SerialName("groupMemberCount") val groupMemberCount: Long,
+    @SerialName("groupInviteCode") val groupInviteCode: String
+)
+
+@Serializable
+data class OngoingAppointmentDto(
+    @SerialName("appointmentId") val appointmentId: Long,
+    @SerialName("appointmentName") val appointmentName: String,
+    @SerialName("availableGroupMemberCount") val availableGroupMemberCount: Long,
+    @SerialName("appointmentTime") val appointmentTime: AppointmentTimeDto
```

Comment:
P1: BaseTimeDto 사용해주세욥
```suggestion
    @SerialName("appointmentTime") val appointmentTime: BaseTimeDto
```